### DISCUSSION TITLE
Lesson 5: Can not update simple storage from 0 to 15 in Brownie

### DISCUSSION THREAD

**Discussion by hre4anyk** [smartcontractkit/full-blockchain-solidity-course-py#1066] [Q&A] [answered]
Fixed it

**Comment by n4n0b1t3** [comment, marked as answer]
Hi, for future reference, please do not remove your original question. Others in our community might have the same issue and your post can potentially help them to fix it. If you just overwrite your post with "Fixed it" only the headline remains and this is not helpful at all :-)

**Reply by cromewar** [reply]
@n4n0b1t3  is right, please for further problems let them as they are @hre4anyk this might help other community members facing the same issue.

**Reply by hre4anyk** [reply]
@n4n0b1t3 @cromewar  guys,  I do agree. I just did not save my code, that's why It did not work. I wanted to delete the post but unsuccessfully
